test/test_codegen.py

```diff
@@ -42,12 +46,13 @@ def test_generate_mechfile(mechname):
 
 
 @pytest.mark.parametrize("mechname", ["uiuc", "sanDiego"])
-def test_get_rate_coefficients(mechname):
+@pytest.mark.parametrize("np_instance", [np, jnp])
+def test_get_rate_coefficients(mechname, np_instance):
```

Comment:
Can we make it so the tests dynamically detect whether jax is importable and adjust their set of numpys accordingly?

Reply:
Done by 8ccf61b.